lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_remove_testers_action.rb

```diff
@@ -26,11 +26,15 @@ def self.run(params)
           UI.user_error!("A maximum of 1000 testers can be removed at a time.")
         end
 
-        UI.message("⏳ Removing #{emails.count} testers from project #{params[:project_number]}...")
-
-        count = fad_api_client.remove_testers(params[:project_number], emails)
-
-        UI.success("✅ #{count} tester(s) removed successfully.")
+        if blank?(params[:group_alias])
+          UI.message("⏳ Removing #{emails.count} testers from project #{params[:project_number]}...")
+          count = fad_api_client.remove_testers(params[:project_number], emails)
+          UI.success("✅ #{count} tester(s) removed successfully.")
+        else
+          UI.message("⏳ Removing #{emails.count} testers from group #{params[:group_alias]}...")
+          fad_api_client.remove_testers_from_group(params[:project_number], params[:group_alias], emails)
+          UI.success("✅ tester(s) removed successfully.")
```

Comment:
Can we include the count of testers in the success message with `emails.count` or is that number not necessarily accurate?

Reply:
It's not an error to try to remove a tester that wasn't a member of the group. So the number might be inaccurate. Since a number is logged just one line above I think it's fine to omit it here - especially since it can be inaccurate.